### ISSUE TITLE
Fajr and Isha calculation isn't working on latest Ubuntu versions: 18.04

### ISSUE THREAD

**Issue by orasul** [arabeyes-org/itools#1]
Ubuntu 18.04.3 LTS \n \l
Linux ubuntu-bionic 4.15.0-70-generic #79-Ubuntu SMP Tue Nov 12 10:36:11 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

itools 1.0-6
libitl0 0.8.0-2

~/.iprayrc
City: Saint-Petersburg, Russia
Latitude: 59.94528
Longitude: 30.24195
UTC: 3
AngleMethod: 2
Mathhab: 1
OffsetList: 1 2 5 0 5 5
FajrInterval: 120
IshaInterval: 100

$ ipraytime

Prayer schedule for,
 City             : Saint-Petersburg, Russia
 Latitude         : 059? 56' 43" N
 Longitude        : 030? 14' 31" E
 Angle Method     : University of Islamic Sciences, Karachi (Shaf'i)
 TimeZone         : UTC+3.0
 Qibla            : 165? 46' 57" E of true North

     Date         Fajr    Shorooq   Zuhr     Asr    Maghrib   Isha 
--------------------------------------------------------------------
 [08-12-2019]    99:99     9:43    12:49    13:47    15:54    99:99

Today's Imsaak    :  99:99
Tomorrow's Imsaak :  99:99
Tomorrow's Fajr   :  99:99

Previously I used Ubuntu 14.04 and 16.04 and it worked fine there:
[08-12-2019]     7:45     9:45    12:54    13:47    15:59    17:39
However it was:
itools 1.0-5
libitl0 0.7.0-3build1
Also we can see slight difference in other times.

**Comment by aelmahmoudy** [comment]
It is working fine with me using your iprayrc file.

Did you file a bug in Ubuntu ?

**Comment by orasul** [comment]
> It is working fine with me using your iprayrc file.
Are You using:
itools 1.0-6
libitl0 0.8.0-2 ?

**Comment by aelmahmoudy** [comment]
No, I actually built itools & itl 0.8.0 manually on a Debian stretch 
machine, and this worked fine.

I just tested now on an Ubuntu 18.04 machine using pre-built packages 
you mentioned, and got the same error.

-- 
‎أحمد المحمودي (Ahmed El-Mahmoudy)
 Digital design engineer
GPG KeyIDs: 4096R/A7EF5671 2048R/EDDDA1B7
GPG Fingerprints:
 6E2E E4BB 72E2 F417 D066  6ABF 7B30 B496 A7EF 5761
 8206 A196 2084 7E6D 0DF8  B176 BC19 6A94 EDDD A1B7

**Comment by aelmahmoudy** [comment]
This needs to be reassigned to ITL project

**Comment by aelmahmoudy** [comment]
This problem is specific to Ubuntu, I just tried the same packages on 
Debian sid, and they gave proper results
